### PR TITLE
DBZ-751 Propagating DECIMAL column precision to Avro schemas

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.jboss.org/browse/DBZ-751

@jpechane This one came up recently in the chat room.

I also did a (manual) test that the parameter from the Connect schema is correctly translated into its Avro schema counterpart.

### CODE REVIEW

File: debezium-connector-mysql/src/test/resources/ddl/decimal_column_test.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-- ----------------------------------------------------------------------------------------------------------------
+-- DATABASE:  decimal_column_test
+-- ----------------------------------------------------------------------------------------------------------------
+
+CREATE TABLE dbz_751_decimal_column_test (
+      id INT AUTO_INCREMENT NOT NULL,
+      rating1 DECIMAL,
+      rating2 DECIMAL(8,4),
+      PRIMARY KEY (id)
+) DEFAULT CHARSET=utf8;
```
Comment (jpechane) [inline debezium-connector-mysql/src/test/resources/ddl/decimal_column_test.sql]: Could you please also verify cases `DECIMAL(8)` and `DECIMAL(8,0)` and all cases also for `NUMERIC` datatype?

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java [hunk 2]
```diff
@@ -182,16 +182,16 @@
         final Struct nvs_int = new Struct(VariableScaleDecimal.schema());
         nvs_int.put("scale", 0).put("value", new BigDecimal("22").unscaledValue().toByteArray());
         final List<SchemaAndValueField> fields = new ArrayList<SchemaAndValueField>(Arrays.asList(
-                new SchemaAndValueField("d", Decimal.builder(2).optional().build(), new BigDecimal("1.10")),
-                new SchemaAndValueField("dzs", Decimal.builder(0).optional().build(), new BigDecimal("10")),
+                new SchemaAndValueField("d", Decimal.builder(2).parameter("connect.decimal.precision", "3").optional().build(), new BigDecimal("1.10")),
```
Comment (jpechane) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java]: Could you please make `connect.decimal.precision` a constant?

**Comment by jpechane** [comment]
@gunnarmorling I think it is fine, thanks. I've just added a request for small changes in the test. One more question - have you verify that the resulting serialized "Avro" has the precision set as requested?

**Comment by gunnarmorling** [comment]
Thanks for reviewing, @jpechane. I've pushed a commit to address your remarks and rebased to latest master.

File: debezium-connector-mysql/src/test/resources/ddl/decimal_column_test.sql (hunk 1 above)
Comment (jpechane) [reply debezium-connector-mysql/src/test/resources/ddl/decimal_column_test.sql]: @gunnarmorling It is fine now, thanks. Could you please add also NUMERIC, NUMERIC(8, 4) etc.? They should be handled in the same way but I've already seen a difference in their handling in the past so I'd say better say than sorry.

**Comment by gunnarmorling** [comment]
@jpechane Pushed another commit to add the same tests for NUMERIC. Regarding

> One more question - have you verify that the resulting serialized "Avro" has the precision set as requested?

Yes, I've manually tested that the field shows up with the correct precision in the schema registry.

**Comment by jpechane** [comment]
@gunnarmorling Applied, thanks!

**Pull request merged**